vtr_flow/parse/parse_config/vpr_noc.txt

```diff
@@ -0,0 +1,15 @@
+#Metrics for the standard VTR flow, which involves both minimum channel
+#width search and relaxed channel width routing
+
+%include "common/vtr_flow.txt"
+
+%include "common/vtr_benchmarks.txt"
+
+%include "common/vpr.common.txt"
+%include "timing/vpr.pack.txt"
+%include "timing/vpr.place.txt"
+%include "timing/vpr.route_min_chan_width.txt"
+%include "timing/vpr.route_relaxed_chan_width.txt"
+
+NoC_agg_bandwidth;vpr.out;NoC Placement Costs. noc_aggregate_bandwidth_cost: (.*), noc_latency_cost: .*,
+NoC_latency;vpr.out;NoC Placement Costs. noc_aggregate_bandwidth_cost: .*, noc_latency_cost: (.*),
```

Comment:
Probably should also add a parse rule for latency constraints met or not (or how many failed) and some test for that case (constraints we can meet and check we do meet them).